src/si/torque_magnitude.rs

```diff
@@ -0,0 +1,155 @@
+//! Torque (aka moment of force) (base unit newton meter, kg<sup>1</sup> · m<sup>2</sup> ·
```

Comment:
List base quantities in the same order as specified in the `system!` macro: length, mass, time, .... Superscript isn't necessary when equal to `1`.

Reply:
Shall I update Force as well?  (that's where I copy pasted from)

Reply:
Added #127 regarding units. Reply there / submit a new PR if you're interested. Otherwise I'll do in the next few days. We'll also want to make kg superscript consistent with that issue. Leaning towards leaving the 1 right now.